kiva/fonttools/font_manager.py

```diff
@@ -290,6 +329,25 @@ def __repr__(self):
         return result
 
 
+class FontSpec(object):
```

Comment:
Maybe make this a private class - i don't think there are users of the class outside this module or outside `kiva.fonttools.*`

Reply:
Good point